deploy/vagrant/provision/roles/tachyon/tasks/config.yml

```diff
@@ -18,6 +18,14 @@
   script: zookeeper.sh
   when: tachyon_masters > 1
 
+- name: set journal folder as a shared folder in underfs for fault tolerance
+  script: set_journal_folder.sh
+  when: tachyon_masters > 1
```

Comment:
Then why not let default value of journal folder be ${tachyon.underfs.address}/journal instead of current ${tachyon.home}/journal?